lang/es.po

```diff
@@ -0,0 +1,96 @@
+msgid ""
+msgstr ""
+"Language: es\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "es translation by Mr. Outis\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d paquete libre\n"
+msgstr[1] "%d paquetes libres\n"
+
+msgid "NONFREE_PACKAGES_COUNT\n"
+msgid_plural "NONFREE_PACKAGES_COUNT\n"
+msgstr[0] "%d paquete privado\n"
+msgstr[1] "%d paquetes privados\n"
+
+msgid "RMS_HAPPY\n"
+msgstr "Sólo paquetes libres - rms estaría orgulloso!\n"
+
+msgid "RMS_DISAPPOINTED\n"
+msgstr "Más de 10%% de paquetes privados. No aprecias tu libertad?\n"
+
+msgid "HELP_USAGE\n"
+msgstr "Uso: vrms-rpm [opciones]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Muestra arte ASCII de rms cuando no se encuentra ningún paquete privado,\n"
+       "    o cuando los paquetes privados son un 10%% o más del total.\n"
+
+msgid "HELP_OPTION_COLOUR\n"
+msgstr "    Controla si la terminal debería de usar sequencias de escape\n"
+       "    para imprimir colores. El predefinido es 'auto', que utiliza colores\n"
+       "    cuando se imprime en la terminal, sólo si no se redirecciona a un archivo o a un pipe.\n"
```

Comment:
This message should not go over 80 characters per line. You can reword it to be shorter, or just add another linebreak (`\n`).